tests/experimental/wgit/test_api.py

```diff
@@ -60,7 +59,7 @@ def test_api_add(capsys, repo):
     repo.add("checkpoint_0.pt")
 
     sha1_hash = repo._sha1_store.get_sha1_hash(chkpt0)
-    with open(os.path.join(".wgit", "checkpoint.pt"), "r") as f:
+    with open(os.path.join(".wgit", "checkpoint_0.pt"), "r") as f:
```

Comment:
super nice check to make sure filename matches.

perhaps the file name can be a bit more random, like "checkpoint_{randint()}.pt"?